.env.alfajores

```diff
@@ -11,4 +11,5 @@ APP_STORE_ID=1482389446
 APP_DISPLAY_NAME=Alfajores
 IOS_GOOGLE_SERVICE_PLIST=GoogleService-Info.alfajores.plist
 DYNAMIC_DOWNLOAD_LINK=https://vlra.app/invite
+DYNAMIC_LINK_DOMAIN_URI_PREFIX=https://vlra.app
```

Comment:
i know that `DYNAMIC_DOWNLOAD_LINK` lives here already....but it feels weird that we have an env variable that is not going to be different between envs? if it's going to be bundled with the app anyway, should we just have the value of this in config.json and not in the env files as well?